sys/picolibc_syscalls_default/syscalls.c

```diff
@@ -187,6 +187,8 @@ static int picolibc_put(char c, FILE *file)
 {
     (void)file;
 
+    if (c == '\n')
+        picolibc_put('\r', file);
```

Comment:
We don't do this for newlib either (and the default terminal accepts just `\n`), so you can drop this.

Reply:
done.